src/java.compiler/share/classes/javax/lang/model/util/Elements.java

```diff
@@ -778,6 +778,16 @@ default TypeElement getOutermostTypeElement(Element e) {
      *                           elements.getTypeElement("I"));
      * }
      *
+     * @apiNote The notion of overriding, by itself, does not involve the
+     * method's return type, listed exceptions, or, to some extent, access
+     * modifiers. These are additional requirements checked by the compiler
+     * (see {@jls 8.4.8.3}).
```

Comment:
Nit: use something like

`(see JLS {@jls 8.4.8.3}).`

as currently written, this would just render a section number pointing to a page.